### PR TITLE
BUG-99486 Support for SUSE12 SP3

### PR DESCRIPTION
**Pull request by lacikaaa**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -130,6 +131,16 @@ build-aws-rhel7:
 	SALT_REPO_FILE="salt-repo-el7.repo" \
 	./scripts/packer.sh build -only=aws-rhel7 $(PACKER_OPTS)
 
+build-aws-sles12sp3:
+	$(ENVS) \
+	AWS_AMI_REGIONS="$(AWS_AMI_REGIONS)" \
+	OS=sles12sp3 \
+	OS_TYPE=sles12sp3 \
+	ATLAS_ARTIFACT_TYPE=amazon \
```
Comment (keyki) [inline Makefile]: incorrect type

File: saltstack/base/salt/dhcp/etc/NetworkManager/dispatcher.d/90-unbound [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 source /etc/dhcp/dhclient-enter-hooks
 
+echo "unbound $@" | systemd-cat -t "dhcp-hook"
```
Comment (keyki) [inline saltstack/base/salt/dhcp/etc/NetworkManager/dispatcher.d/90-unbound]: what is this needed for?

File: saltstack/base/salt/prerequisites/pip.sls [hunk 3]
```diff
@@ -30,3 +32,10 @@ pip_install_requests_security:
 pip_install_virtualenvwrapper:
   pip.installed:
     - name: virtualenvwrapper
+
+install_jq:
```
Comment (keyki) [inline saltstack/base/salt/prerequisites/pip.sls]: is jq install required to move ?

File: saltstack/base/salt/dhcp/etc/NetworkManager/dispatcher.d/90-unbound (hunk 2 above)
Comment (lacikaaa) [reply saltstack/base/salt/dhcp/etc/NetworkManager/dispatcher.d/90-unbound]: I will remove. Leftover from debugging

File: saltstack/base/salt/prerequisites/pip.sls (hunk 3 above)
Comment (lacikaaa) [reply saltstack/base/salt/prerequisites/pip.sls]: there was an issue with salt and python, so I had to move

**Pull request merged**